analysis.go

```diff
@@ -24,83 +27,22 @@ type Info struct {
 
 // Result is a result of a single check, it can have multiple Event and Info
 // objects attached to it.
+// REVIEW: why do we have YAML annotations?
```

Comment:
@philbrookes question for you, why do we have these YAML tags in the structs (Result, Events)?